tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -2947,6 +2948,32 @@ public void handleEnd() {
                         }
                     }
                 });
+
+        fixList.add(
+                'D',
+                "Downgrade paths",
+                new CLDRFilter() {
+                    @Override
+                    public void handlePath(String xpath) {
+                        String value = cldrFileToFilter.getStringValue(xpath);
+                        if (!DowngradePaths.lookingAt(getLocaleID(), xpath, value)) {
+                            return;
+                        }
+                        String fullPath = cldrFileToFilter.getFullXPath(xpath);
+                        XPathParts fullParts = XPathParts.getFrozenInstance(fullPath);
+                        String oldDraft = fullParts.getAttributeValue(-1, "draft");
+                        if (oldDraft != null) {
+                            DraftStatus oldDraftEnum = DraftStatus.forString(oldDraft);
```

Comment:
```suggestion
                        DraftStatus oldDraftEnum = DraftStatus.forXpath(fullPath);
                        if (oldDraftEnum != null) {
```

Reply:
I thought of this, but I think most of the time it is not going to be draft, so because we'll need the fullParts anyway below it...